### PR TITLE
Fixed Gloo on Nomad doc and example

### PR DESCRIPTION
**Pull request by scranton**
Updated documentation and install example for Gloo on Nomad
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/2157

### CODE REVIEW

File: changelog/v1.3.4/fix-nomad-install.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: FIX
```
Comment (kdorosh) [inline changelog/v1.3.4/fix-nomad-install.yaml]: docs changes should be NON_USER_FACING

Comment (kdorosh) [reply changelog/v1.3.4/fix-nomad-install.yaml]: we don't want this rendered in the changelog

**Comment by scranton** [comment]
This PR includes doc AND example code fixes. The example code lives in
github.com/solo-io/gloo/install/nomad and is referenced by the doc. How do
we want to change changelogs for those situations?

**Comment by kdorosh** [comment]
@scranton i deleted my comment because i think `FIX` is correct given the changes to the code example

File: docs/content/installation/gateway/nomad/_index.md [hunk 2]
```diff
@@ -174,11 +178,9 @@ Once you have finished launching these services, you are now ready to install Gl
```
Comment (ned1313) [inline docs/content/installation/gateway/nomad/_index.md]: The separate paths for Linux and macOS are now gone. We should remove this reference.

File: docs/content/installation/gateway/nomad/_index.md [hunk 3]
```diff
@@ -206,39 +208,17 @@ nomad job status gloo
 
 When the deployment is complete, you are ready to [deploy the Pet Store application](#deploying-a-sample-application) to demonstrate Gloo's capabilities.
```
Comment (ned1313) [inline docs/content/installation/gateway/nomad/_index.md]: The lines above are referring to the variables/variables-linux.yaml file. It's just a variable.yaml file now if I am reading this correctly.

File: install/nomad/Vagrantfile [hunk 4]
```diff
@@ -26,84 +27,84 @@ sudo apt-get -y install unzip curl vim
...
+[Install]
+WantedBy=multi-user.target
 EOF
 ) | sudo tee /etc/systemd/system/vault.service
 sudo systemctl enable vault.service
 sudo systemctl start vault
 
 echo "Installing Glooctl..."
-GLOO_VERSION=0.18.2
+GLOO_VERSION=1.2.15
```
Comment (ned1313) [inline install/nomad/Vagrantfile]: Should we try to bump to a 1.3.x release? It does require creating the gateway-proxy and gateway-proxy-ssl objects manually through consul's CLI.

File: install/nomad/demo-local.sh [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+  token: root
+
+gloo:
+  # the port where Gloo serves config to Envoy
+  xdsPort: 9977
+  image:
+    registry: quay.io/solo-io
+    repository: gloo
+    tag: ${GLOO_VERSION}
+  cpuLimit: 1000
```
Comment (ned1313) [inline install/nomad/demo-local.sh]: May want to bump this down to 500. I ran into an issue running it at 1000 on my Linux laptop.

File: install/nomad/jobs/gloo.nomad [hunk 6]
```diff
@@ -421,12 +417,12 @@ EOF
...
-        type = "tcp"
-        interval = "10s"
-        timeout = "2s"
-      }
+      // check {
+      //   name = "alive"
+      //   type = "tcp"
+      //   interval = "10s"
+      //   timeout = "2s"
+      // }
```
Comment (ned1313) [inline install/nomad/jobs/gloo.nomad]: The http check should be commented out too.

**Comment by ned1313** [comment]
I have updated the docs and scripts, and validated them on Linux (Ubuntu 19.10). Someone will need to validate for macOS, since I do not have a Mac available to me.

**Pull request merged**